packages/resolver/src/adapters/api.ts

```diff
@@ -87,8 +91,8 @@ const safelyExecuteOperation = async (
   try {
     const address = await operationFn(resolve, fetcherConfig)
     return {error: null, address, nameServer}
-  } catch (error) {
-    return {error: (error as Error).message, address: null, nameServer}
```

Comment:
We need to identify the errors. 

I was hesitating between this or error codes.

Please let me know what you think